components/tools/OmeroM/src/libs

```diff
@@ -0,0 +1 @@
+../target/libs
```

Comment:
what is this for?

Reply:
Argh, accidentally commited, will remove again (symlink to libs, needed locally to get the matlab stuff running).